src/data/balance/signature-species.ts

```diff
@@ -11,87 +11,87 @@ export type SignatureSpecies = {
+  BRYCEN: [Species.CRYOGONAL, Species.VANILLITE, Species.CUBCHOO, Species.GALAR_DARUMAKA],
+  DRAYDEN: [Species.AXEW, Species.DRUDDIGON, Species.TRAPINCH, Species.DEINO],
+  MARLON: [Species.FRILLISH, Species.TIRTOUGA, Species.WAILMER, Species.MANTYKE],
   // Gym Leaders- Kalos
-  VIOLA: [Species.SURSKIT, Species.SCATTERBUG],
-  GRANT: [Species.AMAURA, Species.TYRUNT],
-  KORRINA: [Species.HAWLUCHA, Species.LUCARIO, Species.MIENFOO],
-  RAMOS: [Species.SKIDDO, Species.HOPPIP, Species.BELLSPROUT],
-  CLEMONT: [Species.HELIOPTILE, Species.MAGNEMITE, Species.EMOLGA],
-  VALERIE: [Species.SYLVEON, Species.MAWILE, Species.MR_MIME],
-  OLYMPIA: [Species.ESPURR, Species.SIGILYPH, Species.SLOWKING],
-  WULFRIC: [Species.BERGMITE, Species.SNOVER, Species.CRYOGONAL],
+  VIOLA: [Species.SCATTERBUG, Species.SURSKIT, Species.CUTIEFLY, Species.DWEBBLE],
+  GRANT: [Species.TYRUNT, Species.AMAURA, Species.BINACLE, Species.HISUI_AVALUGG],
+  KORRINA: [Species.LUCARIO, Species.HAWLUCHA, Species.MIENFOO, Species.PANCHAM],
+  RAMOS: [Species.SKIDDO, Species.HOPPIP, Species.BELLSPROUT, [Species.PHANTUMP, Species.PUMPKABOO]],
+  CLEMONT: [Species.HELIOPTILE, Species.MAGNEMITE, Species.DEDENNE, Species.SHINX],
+  VALERIE: [Species.SYLVEON, Species.MAWILE, [Species.SPRITZEE, Species.SWIRLIX], Species.MR_MIME],
+  OLYMPIA: [Species.ESPURR, Species.SIGILYPH, Species.INKAY, Species.SLOWKING],
+  WULFRIC: [Species.BERGMITE, Species.SNOVER, Species.CRYOGONAL, Species.AMAURA],
   // Gym Leaders- Galar
-  MILO: [Species.GOSSIFLEUR, Species.APPLIN, Species.BOUNSWEET],
-  NESSA: [Species.CHEWTLE, Species.ARROKUDA, Species.WIMPOD],
-  KABU: [Species.SIZZLIPEDE, Species.VULPIX, Species.TORKOAL],
-  BEA: [Species.GALAR_FARFETCHD, Species.MACHOP, Species.CLOBBOPUS],
-  ALLISTER: [Species.GALAR_YAMASK, Species.GALAR_CORSOLA, Species.GASTLY],
-  OPAL: [Species.MILCERY, Species.TOGETIC, Species.GALAR_WEEZING],
-  BEDE: [Species.HATENNA, Species.GALAR_PONYTA, Species.GARDEVOIR],
-  GORDIE: [Species.ROLYCOLY, Species.STONJOURNER, Species.BINACLE],
-  MELONY: [Species.SNOM, Species.GALAR_DARUMAKA, Species.GALAR_MR_MIME],
-  PIERS: [Species.GALAR_ZIGZAGOON, Species.SCRAGGY, Species.INKAY],
-  MARNIE: [Species.IMPIDIMP, Species.PURRLOIN, Species.MORPEKO],
-  RAIHAN: [Species.DURALUDON, Species.TURTONATOR, Species.GOOMY],
+  MILO: [Species.GOSSIFLEUR, Species.APPLIN, Species.SEEDOT, Species.LOTAD],
+  NESSA: [Species.CHEWTLE, Species.ARROKUDA, Species.WIMPOD, Species.MAREANIE],
+  KABU: [Species.SIZZLIPEDE, Species.VULPIX, Species.GROWLITHE, Species.TORKOAL],
+  BEA: [Species.MACHOP, Species.GALAR_FARFETCHD, Species.CLOBBOPUS, Species.FALINKS],
+  ALLISTER: [Species.GASTLY, Species.GALAR_YAMASK, Species.GALAR_CORSOLA, Species.SINISTEA],
+  OPAL: [Species.MILCERY, Species.GALAR_WEEZING, Species.TOGEPI, Species.MAWILE],
+  BEDE: [Species.HATENNA, Species.GALAR_PONYTA, Species.GARDEVOIR, Species.SYLVEON],
+  GORDIE: [Species.ROLYCOLY, Species.STONJOURNER, Species.BINACLE, Species.LARVITAR],
```

Comment:
```suggestion
  GORDIE: [Species.ROLYCOLY, Species.STONJOURNER, [Species.BINACLE, Species.SHUCKLE], Species.LARVITAR],
```
Shuckle is used in all his battles, deserves a shout